google/cloud/spanner/benchmarks/README.md

```diff
@@ -57,13 +55,10 @@ If you are running the benchmarks in a virtual machine, the library can
 automatically use the GCE instance service account (when you do **not** set
 `GOOGLE_APPLICATION_CREDENTIALS`). You may need to grant this service account
 permissions to work with Cloud Spanner. Examine the
-[spanner roles][spanner-roles-link] to chose a role for this account, the
+\[spanner roles\]\[spanner-roles-link\] to chose a role for this account, the
```

Comment:
Another case of changing from a link to verbatim text

Reply:
Another case of the link being broken before.  I'll come back to fix these.